guides/remaining-platform-agnostic.md

```diff
@@ -0,0 +1,85 @@
+# Remaining Platform Agnostic
```

Comment:
"Writing Cross-Platform JavaScript" might be a clearer title, or something along those lines? Thanks for adding this, now we have something to point to.

Reply:
Updated!